src/OidcClientSettings.ts

```diff
@@ -92,6 +92,11 @@ export interface OidcClientSettings {
      * Otherwise, they are added to an array as distinct objects for the claim type. (default: false)
      */
     mergeClaims?: boolean;
+    /**
+     * Indicates if the library should use the legacy merge behavior, which mutates string claims into arrays whenever the data is updated on the remote.
+     * This behavior is enabled by default on v2 and will be removed on v3, since it's not a deterministic way of handling claims.
+     */
```

Comment:
- add  `(default: true)` at the end, like all other props have
- rename `legacyMergeClaimsBehavior` to `mergeClaimsLegacyBehavior`

Reply:
Fixed